sisyphus/son_of_grid_engine.py

```diff
@@ -52,7 +52,7 @@ def try_to_multiply(y, x, backup_value=None):
 
 class SonOfGridEngine(EngineBase):
 
-    def __init__(self, default_rqmt, gateway=None, auto_clean_eqw=True, ignore_jobs=None):
+    def __init__(self, default_rqmt, gateway=None, auto_clean_eqw=True, ignore_jobs=None, pe_name="mpi"):
```

Comment:
Note: This default is somewhat i6 specific, because we configured the PE name to this. However, as i6 is the only real user of SGE, I would argue this is fine?

Reply:
agreed, this is a reasonable default. All other users can still specify their own name if need be.